cl/scrapers/management/commands/clone_from_cl.py

```diff
@@ -710,30 +681,25 @@ def clone_recap_documents(
     return created_recap_documents
 
 
-def clone_tag(
-    session: Session, tag_ids: list, object_type="search.Tag"
-) -> list:
+def clone_tag(session: Session, tag_ids: list) -> list:
     """Clone tags from docket entries or recap documents
 
     :param session: a Requests session
     :param tag_ids: list of tag ids to clone
-    :param object_type: Tag app name with model name
     :return:
     """
     created_tags = []
     for tag_id in tag_ids:
         print(f"Cloning tag id: {tag_id}")
 
-        model = apps.get_model(object_type)
-
         try:
-            tag = model.objects.get(pk=tag_id)
+            tag = Tag.objects.get(pk=tag_id)
```

Comment:
You forgot to import the Tag model from cl.search.models

Reply:
> The creation of opinions was also refactored and a new function called clone_opinion was added.

lol, I looked at the code first and was kicking myself for having missed that.

Good catch on `Tag`.  I thought I would have exercised that when cloning `Docket`s, but I guess none of mine had them.  I also thought `ruff --check` catch it, but I guess not.  I'll start running checking with `flake8`.

Thanks for the review, great suggestions, all implemented.